build.sh

```diff
@@ -46,11 +51,18 @@ do
             ALLYES='true'
             shift
             ;;
+        -j)
+            shift
+            GCC_J=$1
+			shift
+			;;
```

Comment:
Einrückungen bitte wie im Rest des Skriptes, in dem Fall hier also mit Leerzeichen, nicht Tabs.